receiver/stanzareceiver/receiver.go

```diff
@@ -34,12 +44,51 @@ var _ component.LogsReceiver = (*stanzareceiver)(nil)
 
 // Start tells the receiver to start
 func (r *stanzareceiver) Start(ctx context.Context, host component.Host) error {
-	// TODO
-	return nil
+	r.Lock()
+	defer r.Unlock()
+	err := componenterror.ErrAlreadyStarted
+	r.startOnce.Do(func() {
+		err = nil
+		rctx, cancel := context.WithCancel(ctx)
+		r.cancel = cancel
+		r.logger.Info("Starting observiq receiver")
+
+		if obsErr := r.agent.Start(); obsErr != nil {
+			err = fmt.Errorf("start observiq: %s", err)
```

Comment:
```suggestion
			err = fmt.Errorf("start stanza: %s", err)
```

Also should it bother starting the goroutine below if the Start method fails?